CHANGELOG.md

```diff
@@ -19,6 +19,11 @@
   conditional statement.  
   [Daniel Metzing](https://github.com/dirtydanee)
   [#1924](https://github.com/realm/SwiftLint/issues/1924)
+  
+* Indentation can now be specified via a configuration file
+  [Noah McCann](https://github.com/nmccann)
+  [RubenSandwich](https://github.com/RubenSandwich)
```

Comment:
Thanks @RubenSandwich for the original PR on this, and the detailed breakdown on the next steps necessary - that coupled with @jpsim's suggestions helped this come to fruition.